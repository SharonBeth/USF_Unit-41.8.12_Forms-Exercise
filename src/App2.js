import './App.css';
import TodoList from './TodoList';



function App2() {
    return (
        <div className="App2">
            <TodoList />
        </div>
    )
}


export default App2;