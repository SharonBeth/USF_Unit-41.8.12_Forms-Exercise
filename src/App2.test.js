import { render } from '@testing-library/react';
import App2 from './App2';

test('renders App2', () => {
    render(<App2 />);
});

it("matches snapshot", function () {
    const { asFragment } = render(<App2 />);
    expect(asFragment()).toMatchSnapshot();
});