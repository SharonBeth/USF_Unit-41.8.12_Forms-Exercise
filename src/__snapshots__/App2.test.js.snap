// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="App2"
  >
    <div
      class="TodoList"
    >
      <h3>
         Todo List
      </h3>
      <div>
        <form>
          <label
            for="task"
          >
            Task:
          </label>
          <input
            id="task"
            name="task"
            type="text"
            value=""
          />
          <button>
            Add a todo!
          </button>
        </form>
      </div>
      <div />
    </div>
  </div>
</DocumentFragment>
`;
